wcbench.sh

```diff
@@ -480,11 +545,11 @@ uninstall_odl()
 {
     if [ -d $ODL_DIR ]; then
         echo "Removing $ODL_DIR"
-        rm -rf $ODL_DIR
+        sudo rm -rf $ODL_DIR
```

Comment:
In my lappy, ODL works with sudo. So just thought to use sudo to be on same side and for those who need sudo while working with ODL.